index.bs

```diff
@@ -1950,6 +2002,91 @@ The [=remote end steps=] with |session| and <var ignore>command parameters</var>
 
 </div>
 
+#### The browser.createUserContext Command #### {#command-browser-createUserContext}
+
+The <dfn export for=commands>browser.createUserContext</dfn> command creates a
+[=user context=].
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      browser.CreateUserContext = (
+        method: "browser.createUserContext",
+        params: EmptyParams,
+      )
+      </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl local-cddl">
+      browser.createUserContextResult = {
+        userContext: browser.UserContext
+      }
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for browser.createUserContext">
+
+The [=remote end steps=] are:
+
+1. Let |user context| be a new [=empty user context=].
+
+1. Let |result| be a [=/map=] matching the
+   <code>browser.createUserContextResult</code> production with the
+   <code>userContext</code> field set to |user context|'s [=user context id=].
+
+1. Return [=success=] with data |result|.
+
+</div>
+
+#### The browser.removeUserContext Command #### {#command-browser-removeUserContext}
+
+The <dfn export for=commands>browser.removeUserContext</dfn> command closes a
+user context and all browsing contexts in it without running
+<code>beforeunload</code> handlers.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      browser.RemoveUserContext = (
+        method: "browser.removeUserContext",
+        params: {
+          userContext: browser.UserContext
+        },
+      )
+      </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+      <pre class="cddl">
+      EmptyResult
+      </pre>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for browser.removeUserContext">
+
+The [=remote end steps=] with |command parameters| are:
+
+1. Let |user context| be |command  parameters|["<code>userContext</code>"].
+
+1. If the [=user context=] with the [=user context id=] equal to |user context|
+   does not exist in the [=remote end=], return [=error=] with [=error code=] [=no
+   such user context=].
```

Comment:
What happens in case the user tries to remove the "default" user context. Should we allow that, or should we restrict this command to only work with non-default user contexts?

For reference, in Firefox trying to delete the default user context doesn't do anything.

Reply:
Good point, I think we probably want to prevent that given that there could be differences in implementations that are not worth unifying. I think actually in Chrome it would not be possible too.